web/styles/modal.css

```diff
@@ -118,10 +118,7 @@
 .modal__content {
     display: flex;
     flex-direction: column;
-    /* 16px at 14px/1em */
-    font-size: 1.1429em;
     padding: 2px 24px;
```

Comment:
This padding in pixels might want to be in `em`. Note also the `max-width` in this block that seems coupled to this.